WrathCombo/Combos/PvE/BRD/BRD.cs

```diff
@@ -419,14 +422,14 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                                     if (songTimerInSeconds <= 3 && ActionReady(ArmysPaeon))
                                     {
                                         // Special case for Empyreal Arrow: it must be cast before you change to it to avoid drift!
-                                        if (ActionReady(EmpyrealArrow))
+                                        if (ActionReady(EmpyrealArrow) && hasTarget)
                                             return EmpyrealArrow;
                                         return ArmysPaeon;
                                     }
                                 }
                             }
 
-                            if (songArmy && canWeaveDelayed)
+                            if (songArmy && (canWeaveDelayed || !hasTarget))
```

Comment:
Similar to my comment below, I don't think that target checking belongs against a `canWeave` check - this also seems like it would let me clip GCDs if I un-targeted the boss.